app/src/main/res/drawable/buttonbackground.xml

```diff
@@ -3,4 +3,4 @@
     android:shape="rectangle" android:padding="10dp">
     <solid android:color="#fff"/>
     <corners android:radius="46dp"/>
-</shape>
\ No newline at end of file
+</shape>
```

Comment:
all the rest of the drawables use this style

button_background.xml instead of buttonbackground.xml